pkg/actors/actors.go

```diff
@@ -1160,6 +1161,28 @@ func (a *actorsRuntime) doExecuteReminderOrTimer(ctx context.Context, reminder *
 	return nil
 }
 
+func scheduleFromPeriod(period string) (*string, *uint32, error) {
+	if len(period) == 0 {
+		return nil, nil, nil
+	}
+
+	years, months, days, duration, repetition, err := kittime.ParseDuration(period)
+	if err != nil {
+		return nil, nil, fmt.Errorf("unsupported period format: %s", period)
+	}
+
+	if years > 0 || months > 0 || days > 0 {
```

Comment:
Why is this check needed? Wont they all be greater than 0?

Reply:
@cicoyle  AFAICT, Period does not support any point in time due times

Reply:
oh right... the format can be: 
```
time.Duration format, e.g. 2h30m
[ISO 8601 duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) format, e.g. PT2H30M, R5/PT1M30S
```
thats correct, it shouldn't have the yr/mon/day. I'll resolve this 👍🏻